src/coalesce-vue-vuetify3/src/components/input/c-time-picker.vue

```diff
@@ -73,7 +73,7 @@
 }
 
 .c-time-picker__column {
-  max-height: 328px;
+  max-height: 365px;
```

Comment:
Seems like this is only used for the picker, so figured it would be okay to simply increase this height to deal with the new button. I could also remove showing adjacent months and give the today button negative padding?

Reply:
It'll need to be conditional on whether the Today button has been enabled or not. c-datetime-picker should own this CSS though, since the adjustment serves the date picker's needs, not the time picker.

Reply:
My CSS skills are failing me, I can't seem to get this working.

```
  .today-btn .c-time-picker__column {
    max-height: 365px;
  }
```

I figured I could add the above CSS to the `c-datetime-picker` styles and then conditionally apply the `.today-btn` class, but that doesn't appear to be the case. The style never applies.

Reply:
You can't nest it in `.c-datetime-picker` selector because the contents of v-menus get hoisted to the DOM root (usually, and indeed in this case). It would need to be nested in the `.c-datetime-picker__menu` selector, which represents the menu contents.

` .today-btn .c-time-picker__column` is selecting a time picker column that is a child of a `.today-btn`, which is not a hierarchy that exists.